### PR TITLE
Add logging to triggering logic

### PR DESCRIPTION
**Pull request by stevekuznetsov**
Signed-off-by: Steve Kuznetsov <skuznets@redhat.com>

/assign @cjwagner @BenTheElder @fejta 
/cc @krzyzacy

### CODE REVIEW

File: prow/plugins/trigger/generic-comment.go [hunk 1]
```diff
@@ -177,23 +178,27 @@ func filterPresubmits(filter filter, gitHubClient changesGetter, pr *github.Pull
 			toSkipSuperset = append(toSkipSuperset, presubmit)
 		}
 	}
-	return toTrigger, determineSkippedPresubmits(toTrigger, toSkipSuperset), nil
+	toSkip := determineSkippedPresubmits(toTrigger, toSkipSuperset, logger)
+	logger.WithFields(logrus.Fields{"to-trigger": toTrigger, "to-skip": toSkip}).Debug("Filtered %d jobs, found %d to trigger and %d to skip.", len(presubmits), len(toTrigger), len(toSkip))
```
Comment (cjwagner) [inline prow/plugins/trigger/generic-comment.go]: > prow/plugins/trigger/generic-comment.go:182: Entry.Debug call has possible formatting directive %d

Comment (stevekuznetsov) [reply prow/plugins/trigger/generic-comment.go]: god

**Comment by stevekuznetsov** [comment]
IntelliJ really needs to get `go vet` integrated into their IDE :|

**Comment by stevekuznetsov** [comment]
@cjwagner updated

**Comment by BenTheElder** [comment]
_glares impatiently at tide_

**Comment by cjwagner** [comment]
> glares impatiently at tide

Lol I'm doing the same thing

**Comment by stevekuznetsov** [comment]
My tide wasn't fast enough for me today so I just pushed the button :|

**Comment by stevekuznetsov** [comment]
Is it merging anything?

**Comment by cjwagner** [comment]
> Is it merging anything?

Yes it is:
https://prow.k8s.io/tide-history?repo=kubernetes%2Ftest-infra&branch=master

It looks like maybe Tide should have triggered a batch a little sooner, but it is hard to tell.

**Pull request merged**